src/components/Profile/components/VictoryPromise/components/GraphDisplay.tsx

```diff
@@ -21,9 +21,10 @@ const GraphDisplay = ({ scores }: Props) => {
   Object.entries(scores).forEach((score) => {
     const [key, value] = score as [VictoryPromiseCategory, number];
     const index = GraphOrder[key];
+    const colorHex = (Colors[key].match(/#[A-Fa-f0-9]{6}/) || '')[0];
```

Comment:
Wondering if you should have ... `|| [''])[0]` rather than ... `|| '')[0]`

Reply:
I should! Thank you!